setup.py

```diff
@@ -42,7 +42,7 @@
         "pytest_mock==2.0.0",
         "pyzmq==18.1.0",
         "typeguard==2.5.1",
-        "typing_extensions==3.7.4",
+        "typing_extensions~=3.7.4",
```

Comment:
Need to change for psychopy dependency: "typing_extensions>=3.7.4.3",

Reply:
This should cover that, since `~=` allows for version bumps.

Reply:
"typing_extensions~=3.7.4" failed for me during my tests, as 3.7.4 will be installed by default.

Reply:
Interesting. I didn't run into that, but it could be a non-deterministic install order.
I'll change this to be explicit as you suggested.